packages/iTwinUI-react/src/core/ComboBox/ComboBoxMenu.tsx

```diff
@@ -1,38 +1,107 @@
+/*---------------------------------------------------------------------------------------------
+ * Copyright (c) Bentley Systems, Incorporated. All rights reserved.
+ * See LICENSE.md in the project root for license terms and full copyright notice.
+ *--------------------------------------------------------------------------------------------*/
 import cx from 'classnames';
 import React from 'react';
 import { Menu, MenuProps } from '../Menu';
-import { useSafeContext, useMergedRefs } from '../utils';
+import { Surface } from '../Surface';
+import {
+  useSafeContext,
+  useMergedRefs,
+  useVirtualization,
+  VirtualScrollProps,
+  mergeRefs,
+  getWindow,
+} from '../utils';
 import { ComboBoxStateContext, ComboBoxRefsContext } from './helpers';
 
 type ComboBoxMenuProps = Omit<MenuProps, 'onClick'> &
   React.ComponentPropsWithoutRef<'ul'>;
 
+const VirtualComboboxMenu = ({ children, ...props }: VirtualScrollProps) => {
```

Comment:
Why it doesn't have shadow?

Reply:
Surface was supposed to provide one...

Reply:
Surface doesn't have elevation by default, you need to pass it as a prop

Reply:
oh..good catch! somehow I forgot we reset the css default.. 
Added